__tests__/sass/mdn/_a11y-helpers-mixins-test.scss

```diff
@@ -0,0 +1,15 @@
+@include describe('focus-visible') {
+    @include it(
+        'Returns a box-shadow style that is applied to elements that currently has focus'
+    ) {
+        @include assert {
+            @include output {
+                @include focus-visible();
+            }
+
+            @include expect {
+                box-shadow: 1px 1px 5px #2196f3, -1px -1px 5px #2196f3;
```

Comment:
Is there a way to write these tests that allows you to change the color or size of the shadow without breaking the test?